csp_billing_adapter_local/plugin.py

```diff
@@ -44,6 +46,15 @@ def get_local_path(filename):
     return local_storage_path
 
 
+@csp_billing_adapter.hookimpl(trylast=True)
+def setup_adapter(config: Config):
+    log_to_file = logging.FileHandler(CSP_LOG_FILEPATH)
+    csp_log = logging.getLogger(LOGGER_NAME)
+    csp_log.addHandler(log_to_file)
+    logging_level = config.get('logging', {}).get('level', 'INFO')
```

Comment:
This is fine although duplicative if the core code process the remainder of logging config options. Because the config of parent handler propagates to children.

Reply:
do we want to log to a file _always_ ?
AFAIU, from the conversations, log to a file option should be present on `local` only

Reply:
There's no point to log to a file in a container. This seems fine. But the log level if there's a configuration option would also be set in core adapter code where the basic config is setup.